server/utils/world/getWorldDetails.ts

```diff
@@ -0,0 +1,34 @@
+import { World } from "../topiaInit.js";
+import { errorHandler } from "../errorHandler.js";
+import { initializeWorldDataObject } from "./initializeWorldDataObject.js";
+import { Credentials, DataObjectType } from '../../types/index.js';
+
+type WorldDataObject = {
+  scenes: {
+    [key: string]: DataObjectType
+  },
+}
+
+export const getWorldDetails = async (credentials: Credentials) => {
+  try {
+    const { assetId, urlSlug } = credentials
+    const sceneDropId = credentials.sceneDropId || assetId
+
+    const world = World.create(urlSlug, { credentials });
+    await world.fetchDetails();
```

Comment:
we should use Promise.all here since performance in this method seems to be critical, because it's used often